### PR TITLE
Add some metrics

### PR DESCRIPTION
**Pull request by ivanyu**
- `segment-copy-time-avg`
- `segment-copy-time-max`
- `segment-fetch-rate`

### CODE REVIEW

File: core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManager.java [hunk 1]
```diff
@@ -105,14 +113,29 @@ public RemoteStorageManager() {
...
+        segmentCopyTime.add(metrics.metricName("segment-copy-time-max", metricGroup), new Max());
+
+        segmentFetchPerSec = metrics.sensor("segment-fetch");
+        segmentFetchPerSec.add(metrics.metricName("segment-fetch-rate", metricGroup), new Rate());
+
+        segmentFetchToFirstByteTime = metrics.sensor("segment-fetch-to-first-byte-time");
+        segmentFetchToFirstByteTime.add(metrics.metricName("segment-fetch-to-first-byte-time-avg", metricGroup),
+            new Avg());
+        segmentFetchToFirstByteTime.add(metrics.metricName("segment-fetch-to-first-byte-time-max", metricGroup),
+            new Max());
```
Comment (jeqo) [inline core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManager.java:138]: Could we add some comments (not sure if here or in README maybe?) about how this metric is going to be used? Others are pretty standard, but time to first byte seems to deliver some insights that are not obvious.

**Review by jeqo — COMMENTED** [review]
Looks pretty good, just a comment about time to first byte metric.

File: core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManager.java (hunk 1 above)
Comment (ivanyu) [reply core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManager.java]: I removed this one for now. Need more thinking of how to better implement it and next time I'll add a description

**Review by jeqo — APPROVED** [review]
LGTM

**Pull request merged**